lib/plugins/aws/package/lib/stripNullPropsFromTemplateResources.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  stripNullPropsFromTemplateResources() {
+    const resources = this.serverless.service.provider.compiledCloudFormationTemplate.Resources;
+
+    if (Object.keys(resources).length > 0) {
+      for (const resource of Object.values(resources)) {
+        if (resource.Properties) {
+          for (const [propName, propVal] of Object.entries(resource.Properties)) {
+            if (propVal === null) {
+              delete resource.Properties[propName];
+            }
+          }
+        }
+      }
+    }
+
+    return Promise.resolve();
```

Comment:
I think it's not really needed to return a `Promise` here - it's a sync function and let's treat it this way